.github/CODEOWNERS

```diff
@@ -86,3 +86,5 @@ sycl/doc/extensions/ @intel/dpcpp-specification-reviewers
 
 xpti/ @tovinkere @andykaylor
 xptifw/ @tovinkere  @andykaylor
+
+llvm/tools/sycl-post-link/ @AlexeySachkov
```

Comment:
I think we can also add @kbobrovs to list of sycl-post-link codeowners.
@kbobrovs , WDYT?

Reply:
+1 for adding @kbobrovs. It's recommended to have at least 2 code owners to back up during vacations, etc.

Reply:
sure, you can add me

Reply:
Done